Hearthstone/app/src/main/java/com/example/hearthstone/data/network/api/HearthstoneApi.kt

```diff
@@ -11,20 +11,20 @@ interface HearthstoneApi {
 
     @GET("info")
     suspend fun getCards(
-        @Header("X-RapidAPI-Key") apiKey: String = "",
+        @Header("X-RapidAPI-Key") apiKey: String = "d076ef0d6emsh14986fa05e38ffdp1ea135jsn1e4319697846",
```

Comment:
This is your API key. I don't recommend uploading this to github especially since you're project is public.